cpp/src/dictionary/set_keys.cu

```diff
@@ -163,6 +165,53 @@ std::vector<std::unique_ptr<column>> match_dictionaries(std::vector<dictionary_c
   return result;
 }
 
+/**
+ * @copydoc match_dictionaries(std::vector<table_view>,rmm::mr::device_memory_resource,cudaStream_t)
```

Comment:
No but I thought it would be nice.

Reply:
This doesn't help anyone, though. It doesn't help the human reader, and Doxygen will likely have to ignore it since it already has documentation for this function from the header.